build/configs/rtl8721csm/rtl8721csm_download.sh

```diff
@@ -238,12 +244,10 @@ download_specific_partition()
 	fi
 	echo "============================="
 	exe_name=$(get_executable_name ${parts[$partidx]})
-	./amebad_image_tool $TTYDEV 1 ${offsets[$partidx]} ${exe_name}
+	./amebad_image_tool 1 $TTYDEV 1 ${offsets[$partidx]} ${exe_name}
```

Comment:
Maybe 1 is for flashing, and 2 is for erasing. How about define the named value for readability?

Reply:
OK, I will change that with 'download' and 'erase' for the readability, thanks